src/Components/Map.tsx

```diff
@@ -61,7 +61,7 @@ export const Map = ({ onClickPin, lockMap, viewState, setViewState }: MapProps )
                         mapStyle='mapbox://styles/mapbox/streets-v11'
                         mapboxApiAccessToken={token}
                     >
-                        <div style={{top: 62, left: 15, position: 'absolute', zIndex: 1}}>
+                        <div style={{display: 'none', top: 62, left: 15, position: 'absolute', zIndex: 1}}>
```

Comment:
I think from slack that this was deliberately removed today, and putting it back is maybe a merge artifact?

Reply:
ah, ok - I can fix this